tool/generate-specs.sh

```diff
@@ -14,7 +14,7 @@ function generate_spec() {
 for dir in external/nextcloud-server external/nextcloud-notifications; do
   (
     cd "$dir"
-    composer update
+    composer install
     composer install --no-dev
```

Comment:
Technically only the first on is needed, but it will leave behind some files if you have a different php or composer version. The second command basically cleans up those files. Otherwise we would always have diffs on the submodules which is annoying. The composer update was just wrong because it updated the lockfile which we don't want to do because it could pull in versions that we aren't supposed to use.